src/wp-includes/media.php

```diff
@@ -5455,6 +5455,16 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	/*
+	 * Skip programmatically created images within post content as they need to be handled together with the other
+	 * images within the post content.
+	 * Without this clause, they would already be counted below which skews the number and can result in the first
+	 * post content image being lazy-loaded only because there are images elsewhere in the post content.
+	 */
+	if ( ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context ) && doing_filter( 'the_content' ) ) {
```

Comment:
```suggestion
	if ( in_array( $context, array( 'the_post_thumbnail', 'wp_get_attachment_image' ),  true ) && doing_filter( 'the_content' ) ) {
```
Is this more readable?

Reply:
Either works, and I like your version for readability, though I'm unsure on the performance implications of `in_array` vs a regular comparison with `||`.

How about using an array with keys and `isset()`? That is certainly faster than `in_array()` and still addresses your readability concern. I'll push an update for consideration.